### PR TITLE
Various Fixes

### PR DESCRIPTION
**Pull request by 0xTim**
- **Make company cards link out to new pages**
- **Update links in footer and navbar**
- **Add accessibility labels to company cards**
- **Better span links**
- **NPM updates**
- **Add some accessibility labels**
- **Add a load of accessibility labels**

### CODE REVIEW

File: Sources/VaporDesign/Components/SiteFooter.swift [hunk 1]
```diff
@@ -40,8 +40,7 @@ public struct SiteFooter: Component {
                                         } else if currentSite == .main {
                                             Link("Team", url: "/team").class("nav-link")
                                         } else {
-                                            #warning("Fix link")
-                                            Link("Team", url: "https://www.vapor.codes/").class("nav-link").linkTarget(.blank)
+                                            Link("Team", url: "https://www.vapor.codes/team").class("nav-link").linkTarget(.blank)
```
Comment (ptoffy) [inline Sources/VaporDesign/Components/SiteFooter.swift:43]: I think links should not open in a new tab if they point to the same website? The same goes for all links in the footer here

**Review by ptoffy — CHANGES_REQUESTED** [review]

File: Sources/VaporDesign/Components/SiteFooter.swift (hunk 1 above)
Comment (0xTim) [reply Sources/VaporDesign/Components/SiteFooter.swift:43]: Well technically it's a different site but I'm happy to have it stay in the same tab

Comment (ptoffy) [reply Sources/VaporDesign/Components/SiteFooter.swift:43]: 🤔 isn't it just a different page on the same site?

Comment (ptoffy) [reply Sources/VaporDesign/Components/SiteFooter.swift:43]: Ah you mean because it's shared by the blog too

Comment (0xTim) [reply Sources/VaporDesign/Components/SiteFooter.swift:43]: Yeah if we're on the main site it stays in the same tab, if we're on a different site, like the blog, it opens that page in a new tab

**Review by ptoffy — APPROVED** [review]

**Pull request merged**